server/plugin.go

```diff
@@ -58,9 +60,13 @@ func (p *Plugin) MessageWillBePosted(c *plugin.Context, post *model.Post) (*mode
 			return false
 		case *markdown.ReferenceImage:
 			return false
+		case *markdown.Autolink:
+			textNode = typedNode.Children[0].(*markdown.Text)
```

Comment:
I don't particularly like this line - but the server uses an array for Autolink.Children even though those nodes always have exactly one Text child and nothing else.